README.md

```diff
@@ -16,6 +16,25 @@ Use cases may include:
 - Github (v3 API)
 
 
+<h2>Configuration File</h2>
+Freighter requires the presence of a .freighter file with any of the following supported extensions:
```

Comment:
Can this be optional? I think it's a shame to always require a file on disk to be written before you can use the CLI. - For the use case within a cluster thats populating docker secrets this makes sense.

For other use cases, though, someone may want to run freighter as a docker container but in a cli like fashion, itd be nice to have that as a single line statment without having to write out a file to disk beforehand and worry about formats etc